src/icon.rs

```diff
@@ -30,7 +30,7 @@ impl Icons {
                     Self::get_default_icons_by_name(),
                     Self::get_default_icons_by_extension(),
                     "\u{f016}", // 
-                    "\u{f115}", // 
```

Comment:
do not change the default icon, if want to, we can raise another issue and discuss it together.

Reply:
Okay sounds good. I changed it back, and the tests, too.